src/h_.hpp

```diff
@@ -468,17 +541,12 @@ void h_::mul(bi_t& result, const bi_t& a, const bi_t& b) {
   const size_t m = a.size();
   const size_t n = b.size();
 
-  const auto [n_result_digits, overflow] = uints::uadd_overflow(m, n);
```

Comment:
By the constraints imposed by this library (namely, that `max_digits` is less than or equal to `SIZE_MAX / sizeof(digit)`, this summation will never overflow.